collectd-openstack/README.md

```diff
@@ -0,0 +1,300 @@
+        AuthURL "http://localhost/identity/v3"
+        Username "admin"
+        Password "secret"
+        ProjectName "alt_demo"
+        ProjectDomainId "default"
+        UserDomainId "default"
+  </Module>
+</Plugin>
+```
+
+### USAGE
+
+#### Interpreting Built-in dashboards
+
+- **HYPERVISOR**:
+
+  - **Running VMs**: Shows the number of VMs running in the hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-vms.png' width=200px>](./img/chart-openstack-hypervisor-vms.png)
+
+  - **Load Average**: Shows the average CPU load on the hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-load.png' width=200px>](./img/chart-openstack-hypervisor-load.png)
+
+  - **Memory Usage**: Shows the memory usage free vs used in the hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-memory-usage.png' width=200px>](./img/chart-openstack-hypervisor-memory-usage.png)
+
+  - **Disk Usage**: Shows the disk usage free vs used in the hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-disk-usage.png' width=200px>](./img/chart-openstack-hypervisor-disk-usage.png)
+
+  - **CPUs and VCPUs**: Shows the number of CPUs available and VCPUs used in the hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-cpus-vcpus.png' width=200px>](./img/chart-openstack-hypervisor-cpus-vcpus.png)
+
+  - **Physical CPUs**: Shows the number of pysical cores on the host running hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-physical-cpus.png' width=200px>](./img/chart-openstack-hypervisor-physical-cpus.png)
+
+  - **Total Disk**: Shows the total disk available on the host running hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-total-disk.png' width=200px>](./img/chart-openstack-hypervisor-total-disk.png)
+
+  - **Total Memory**: Shows the total RAM memory available on the host running hypervisor.
+
+    [<img src='./img/chart-openstack-hypervisor-total-memory.png' width=200px>](./img/chart-openstack-hypervisor-total-memory.png)
+
+
+- **TENANT**:
+
+  - **Used Instances**: Shows the number of used instances in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-used-instances.png' width=200px>](./img/chart-openstack-tenant-used-instances.png)
+
+  - **Memory Utilization**: Shows the percentage of utilized memory in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-memory-util.png' width=200px>](./img/chart-openstack-tenant-memory-util.png)
+
+  - **Disk Utilization**: Shows the percentage of utilized disk in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-disk-util.png' width=200px>](./img/chart-openstack-tenant-disk-util.png)
+
+  - **Instances Available vs Used**: Shows the number of available instances and used instances in the tenant/project.
+
+    [<img src='./img/chart-openstack-tenant-instances.png' width=200px>](./img/chart-openstack-tenant-instances.png)
+
+  - **Memory Usage**: Shows the used memory over available in the tenant/project.
```

Comment:
Not sure what "used memory over available" means. Maybe it's clear if I'm looking at the dashboard, but if this is percentage, then something like "percentage of used memory" could be clearer. Same for the following few entries.

Reply:
Updated the metric description accordingly